cve-suppress.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm suppressing some of the vulnerabilities, since newer versions/fixes are not available yet, we need to keep an eye on their release though